mypy.ini

```diff
@@ -0,0 +1,3 @@
+[mypy]
```

Comment:
We prefer tooling configuration to be placed in `pyproject.toml` instead of tool-specific files.

Reply:
as this project is not using `pyproject.toml` I'm going to include in `setup-cfg`